main/background.ts

```diff
@@ -55,6 +55,11 @@ express_app.get("/", (req, res) => {
 	res.send(websockets_client(address))
 })
 
+express_app.get('/images/:imageName', (req, res) => {
+    const imageName = req.params.imageName;
+    res.sendFile(`${__dirname}/public/images/${imageName}`);
```

Comment:
Wiesz co, bo ten folder jest w `.gitignore`, więc nie wysyłasz tych plików na serwer, więc u mnie wygląda to tak 
![image](https://github.com/Zajmujemy-Dzieci/zajmujemy-dzieci/assets/60236390/35ebefc9-d80e-40de-acd1-f8075389eda0)